jobs/config.json

```diff
@@ -4448,16 +4448,18 @@
   },
   "ci-kubernetes-e2e-gci-gke": {
     "args": [
+      "--check-leaked-resources",
+      "--cluster=",
```

Comment:
Sure. We'll probably have some better tools for debugging busted clusters later.